scripts/test_all.sh

```diff
@@ -24,7 +24,7 @@ echo "_____________________________________________ TESTING"
 
 # TEMPORARY FIX: comment out UI tests
 # containers=( kernel ui odk couchdb-sync )
-containers=( kernel odk couchdb-sync )
+containers=( client kernel odk couchdb-sync)
```

Comment:
The logic says that first test kernel and afterwards the modules depending on it ;-)

Reply:
Ah yes, but when I was fighting travis yesterday, I cared not for logic. I just needed the client tests to fail as quickly as possible. I agree though.